composer.json

```diff
@@ -19,44 +19,40 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.4",
-        "yiisoft/yii2": "~2.0.13",
+        "ext-mbstring": "*",
+        "php": "^7.2 || ^8.0",
+        "yiisoft/yii2": "~2.0.16",
         "yiisoft/yii2-bootstrap": "~2.0.0",
-        "phpdocumentor/reflection": "^3.0.1",
-        "phpdocumentor/reflection-docblock": "^2.0.4",
-        "nikic/php-parser": "^1.0",
-        "cebe/js-search": "~0.9.3",
-        "cebe/markdown": "~1.0.0 | ~1.1.0 | ~1.2.0",
-        "cebe/markdown-latex": "~1.0",
-        "scrivo/highlight.php": "~9.13"
+        "phpdocumentor/reflection": "dev-access-phpparser-node",
+        "nikic/php-parser": "^4.0",
+        "cebe/js-search": "~0.9.0",
+        "cebe/markdown": "^1.0",
+        "cebe/markdown-latex": "^1.0",
+        "scrivo/highlight.php": "^9.0"
     },
     "require-dev": {
-        "cweagans/composer-patches": "^1.7",
-        "phpunit/phpunit": "4.8.34"
+        "phpunit/phpunit": "*"
```

Comment:
I think we need to specify versions more explicitly.

Reply:
That's how it was in @bizley's draft (initial commit). Maybe there is a reason for that? Different PHP versions maybe? :thinking: Anyway I agree with you, in terms of dependencies this is not safe. I'll take a look.

Reply:
Yes, I did it like that because we usually don't have constraint on the PHP version. I can see now that since we have it limited we should also limit phpunit but still allowing all range of it for the given php versions.